### PR TITLE
use updated packages

### PR DESCRIPTION
**Pull request by TalDerei**
follow-up from https://github.com/penumbra-zone/web/pull/1768

### CODE REVIEW

**Review by turbocrime — APPROVED** [review]

**Comment by TalDerei** [comment]
cc @turbocrime 

screenshot_2024-09-05_at_11 35 37___pm

**Comment by turbocrime** [comment]
you updated `@bufbuild/protobuf` accross a major version with breaking changes. syncpack is now configured to restrict its version.

unfortunately, some recent changes in the web repo are breaking vitest - the `svgr` tool was added which requires a URL reference to SVG files, and one was applied.

https://github.com/penumbra-zone/web/blob/0558dc2d091df7976bb6838aa0070cff7f1f93cb/apps/minifront/src/components/syncing-dialog/sync-animation.tsx#L1

now wasm imports don't work due to this issue https://github.com/vitest-dev/vitest/issues/2150

i'll try the workaround, and also PR a fix for the web repo.

**Comment by turbocrime** [comment]
getters aren't the problem. example branch

https://github.com/prax-wallet/web/tree/turbocrime/identify-wasm-break

- uses the correct protobufs
- consumes the optional getters refactor

test this branch locally with packages built from https://github.com/penumbra-zone/web/commit/3a5c07438e8f7a48d2eed7e0899a3742072479fd

tests pass

**Comment by turbocrime** [comment]
you also version bumped vitest, that was the source of the problem.

**Comment by turbocrime** [comment]
also RM prax-marketing site #182

File: .syncpackrc [hunk 1]
```diff
@@ -21,6 +21,16 @@
...
+    {
+      "range": "",
+      "dependencies": [
+        "@penumbra-zone/**",
+        "@prax-wallet/**"
+      ],
+      "packages": [
+        "**"
+      ]
+    },
```
Comment (turbocrime) [inline .syncpackrc:33]: this configuration means version specifiers must be exact.

**Comment by TalDerei** [comment]
instead of `pnpm -r update --latest`, the command `pnpm update --latest --save-exact` included breaking version ranges...

**Pull request merged**